scripts/v2_macd_bb_rsi_reversion_multitimeframe.py

```
@@ -0,0 +1,264 @@
+from hummingbot.connector.connector_base import ConnectorBase
+from hummingbot.core.clock import Clock
+from hummingbot.core.data_type.common import OrderType, PositionMode, PriceType, TradeType
+from hummingbot.data_feed.candles_feed.candles_factory import CandlesConfig
+from hummingbot.strategy.strategy_v2_base import StrategyV2Base, StrategyV2ConfigBase
+from hummingbot.strategy_v2.executors.position_executor.data_types import PositionExecutorConfig, TripleBarrierConfig
+from hummingbot.strategy_v2.models.executor_actions import CreateExecutorAction, StopExecutorAction
+
+
+class MACDBBRSIConfig(StrategyV2ConfigBase):
+    script_file_name: str = os.path.basename(__file__)
+    markets: Dict[str, List[str]] = {}
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)